### PR TITLE
Fixed pom and test

### LINKED ISSUE #48: Fix sleep in test to something else (cassandra connector)

### PR DESCRIPTION
**Pull request by mgobec**
Fixed pom and javadoc argument
Fixed test (connection reset error) and removed sleep

This PR fixes #48

### CODE REVIEW

File: cassandra-diagnostics-connector21/pom.xml [hunk 1]
```diff
@@ -1,91 +1,91 @@
...
+                                    <include>net.bytebuddy:*</include>
+                                </includes>
+                            </artifactSet>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
 </project>
```
Comment (nbozic-zz) [inline cassandra-diagnostics-connector21/pom.xml:91]: Can one of you two add this to formatter to have same formatting? Or one should start using formatter

Comment (mgobec) [inline cassandra-diagnostics-connector21/pom.xml:91]: @nivancevic finally started using formatter...

**Comment by nbozic-zz** [comment]
I see in comment of this PR that you removed thread sleep but I do not see that code?

There is sleep method as well which is probably not used when you removed line so remove method as well.

**Comment by nivancevic** [comment]
That's because it is 'removed'

File: cassandra-diagnostics-connector21/src/test/java/io/smartcat/cassandra/diagnostics/connector/ConnectorImplTest.java [hunk 2]
```diff
@@ -37,21 +36,17 @@ public void report(Query query) {
...
 
     @Test
     public void test() {
-        session.execute("CREATE KEYSPACE IF NOT EXISTS test_keyspace " +
-                "WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };");
+        session.execute("CREATE KEYSPACE IF NOT EXISTS test_keyspace "
+                + "WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };");
         session.execute("CREATE TABLE IF NOT EXISTS test_keyspace.test_table (uid uuid PRIMARY KEY);");
         session.execute("SELECT * FROM test_keyspace.test_table");
-        sleep(1000);
```
Comment (mgobec) [inline cassandra-diagnostics-connector21/src/test/java/io/smartcat/cassandra/diagnostics/connector/ConnectorImplTest.java:53]: Here I am..... Rocking like a hurricane

Comment (nbozic-zz) [inline cassandra-diagnostics-connector21/src/test/java/io/smartcat/cassandra/diagnostics/connector/ConnectorImplTest.java:53]: remove `private static void sleep(int duration) {` as well

**Pull request merged**